src/book.py

```diff
@@ -129,10 +130,17 @@ def tokenize(self, on, rem_stopwords=True, stopword_lang='english',
         stopword_lang (default 'english'): language of stopword corpus to use
         add_stopwords CURRENTLY UNWORKING (default []): list of words to be added to stopword list
         """
+
+        if include_punctuation:
+            token = self._text.lower()
+        else:
+            # remove punctuation
+            token = self._text.lower().translate(
+                str.maketrans('', '', string.punctuation + '”“’'))
```

Comment:
Does this only exclude double quotations and apostrophes? There are so many small characters it's a little hard to tell.

Reply:
Take a look at `string.punctuation`. It includes a bunch of other things. But -- don't merge yet. I realized a mistake I have to fix.

Reply:
I see that it already includes quotations. Then what is the extra bits? Are they unique characters?

Reply:
They're technically different characters I think. " instead of ”, etc. If it's redundant, it doesn't matter

Reply:
Are we ready to merge this?